cmd/ltctl/main.go

```diff
@@ -34,7 +34,7 @@ func RunCreateCmdF(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to create SSH agent: %w", err)
 	}
 
-	initData := config.DBDumpURI == ""
+	initData := config.DBDumpURI == "" || config.ExternalDBSettings.DataSource != ""
```

Comment:
Wondering if this should be an invalid case at the config level. Does having both defined make sense?

Reply:
No, because it will try to upload the dump to the database using the `TerraformDBSettings` credentials and getting the database URL from terraform's output `DBWriter`.

Reply:
I don't understand this either. With this change, if both `DBDumpURI and `DataSource` are non-empty, `initData` is true, right? Shouldn't this be
```suggestion
	initData := config.DBDumpURI == "" && config.ExternalDBSettings.DataSource == ""
```

Reply:
That looks better 👍

Reply:
I'd still welcome an addition to `func (c *Config) IsValid()` if having both set is a clear invalid case to avoid unexpected side effects. Not blocking though.

Reply:
https://github.com/mattermost/mattermost-load-test-ng/pull/870/commits/9aa44f4400579c82786440b7f67bae3b3f283823